src/containers/App/App.js

```diff
@@ -6,12 +6,6 @@ import Header from '../../components/Header/Header';
 import ParentSection from '../../components/ParentSection/ParentSection';
 
 export class App extends Component {
-  constructor() {
```

Comment:
Do we need the constructor in order to access lifecycle methods?

Reply:
We should have access to that via { Component } !